CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Dependencies updated in [7718](https://github.com/stripe/stripe-android/pull/771
 * Bumped Kotlin from 1.8.22 to 1.9.21.
 * Bumped Coroutines from 1.6.4 to 1.7.3.
 * Bumped Compose Compiler from 1.4.8 to 1.5.6.
+Dependencies updated in [7851](https://github.com/stripe/stripe-android/pull/7851):
```

Comment:
Nit: We should change this to be `Dependencies updated in 7718 and 7851`, and then list them all.

Reply:
The copy paste gremlins strike again!